libs/workspace/src/builders/elements/build.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BuilderContext, BuilderOutput } from '@angular-devkit/architect';
+import { join } from 'path';
+import { forkJoin, from, Observable, of } from 'rxjs';
+import { catchError, map, mapTo, switchMap, tap } from 'rxjs/operators';
+import { ElementsOptions } from './schema';
+import { tryJsonParse, NxJson } from '@dynatrace/shared/node';
+import { copyRootPackageJson } from '../../utils/copy-root-package-json';
+
+/**
+ * Custom builder for the web-components package builder.
+ * This builder will schedule and run all projects that are tagged
+ * with a given tag in the nx.json. It is primarily used to run and package
+ * the web-components part of the library.
+ */
+export function elementsBuilder(
+  options: ElementsOptions,
+  context: BuilderContext,
+): Observable<BuilderOutput> {
+  const project = context.target !== undefined ? context.target!.project : '';
+  context.logger.info(`Packaging ${project}...`);
+
+  // Get all builds that are tagged with elements
+  context.logger.info('Reading nx.json file...');
+  return from(
+    tryJsonParse<NxJson>(join(context.workspaceRoot, 'nx.json')),
+  ).pipe(
+    // Find the projects that need to be built as part of the elements build.
+    map((nxJson: NxJson) =>
+      filterTaggedProjects(
+        nxJson,
+        options.buildTag || 'scope:elements',
+        context.target!.project,
```

Comment:
Use the project from above

```suggestion
        project,
```